modules/cluster-logging-deploy-cli.adoc

```diff
@@ -332,7 +332,7 @@ spec:
 However, an unmanaged deployment does not receive updates until Cluster Logging is placed back into the `Managed` state.
 <3> Settings for configuring Elasticsearch. Using the Custom Resource (CR), you can configure shard replication policy and persistent storage.
 <4> Specify the number of Elasticsearch nodes. See the note that follows this list.
-<5> Specify that each Elasticsearch node in the cluster is bound to a Persistent Volume Claim.
+<5> Enter the name of an existing StorageClass for Elasticsearch storage. The StorageClass should allocate block storage for best performance.
```

Comment:
> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.

Maybe:
> For best performance, specify a StorageClass that allocates block storage.

Reply:
I was thinking that too. Thanks!

Reply:
We should update this for OSD as well. OSD customers can now utilize AWS or GCP storage depending on their platform. That said, we do have a larger audit/cleanup story to address the issue of where we are applying and managing filtering like this for different distros.